experiments/m3/src/evaluation.py

```diff
@@ -50,11 +50,11 @@ def main(test: bool = False):
     time = evaluation.query('metric=="time"').T
     if test:
         expected_results = {
-            'AutoARIMA': 4.87, 
+            'AutoARIMA': 4.57,
```

Comment:
there were a couple of bugs that impacted the performance, so fixing them reduced this metric but slightly increased the ensemble one (my guess is that some of the errors were actually helping the ensemble)

Reply:
cool